analyzer/src/funTest/assets/projects/external/directories-expected-output.yml

```diff
@@ -31,6 +31,8 @@ analyzer:
       - "Mozilla Public License 2.0"
       declared_licenses_processed:
         spdx_expression: "MPL-2.0"
+        mapped:
+          Mozilla Public License 2.0: "MPL-2.0"
```

Comment:
I'm surprised that even strings containing spaces are not quoted when used as key, but values seem to be always quoted. I wonder what happens if the mapped license strings contains a colon, will it be escaped or the string quoted? Should we configure the mapper to always quote keys with spaces, if possible?

Reply:
>  I wonder what happens if the mapped license strings contains a colon, will it be escaped or the string quoted?

Quoted, see here:
https://github.com/oss-review-toolkit/ort/blob/99b8d760ac2bdbd4ea5b6f6c1b35bb7bdcdeaebd/analyzer/src/funTest/assets/projects/external/git-repo-expected-output.yml#L8903

> Should we configure the mapper to always quote keys with spaces, if possible?

No, I think we should keep it as is and let the mapper add quotes only if required.